force-app/main/default/classes/AccountTriggerUtility.cls

```diff
@@ -0,0 +1,24 @@
+public with sharing class AccountTriggerUtility {
+  
+        public static void afterUpdate(List<Account> accounts){
+
+  
+
+    }
+
+    public static void afterInsert(List<Account> accounts){
```

Comment:
Tutaj nazwy metod nie powinny się tak nazywać, jeżeli chcesz to robić jako Utility, to chociaż metody w środku powinny nazywać się w taki sposób, żeby czytając ich wywołanie już wiedzieć co robią, a tutaj afterInsert nam nic nie mówi, bo kontekst tego że to after Insert mamy juz w AccountTriggerHandlerze. Więc tutaj metoda powinna się nazywać przykładowo "insertContactsUnderAccounts" i wtedy jak czytasz kod to widzisz od razu +/- co metoda robi